packages/lit-analyzer/src/lib/rules/no-unclosed-tag.ts

```diff
@@ -11,18 +34,18 @@ const rule: RuleModule = {
 		priority: "low"
 	},
 	visitHtmlNode(htmlNode, context) {
-		if (!htmlNode.selfClosed && htmlNode.location.endTag == null) {
-			// Report specifically that a custom element cannot be self closing
-			//   if the user is trying to close a custom element.
-			const isCustomElement = isCustomElementTagName(htmlNode.tagName);
-
-			context.report({
-				location: rangeFromHtmlNode(htmlNode),
-				message: `This tag isn't closed.${isCustomElement ? " Custom elements cannot be self closing." : ""}`
-			});
+		if (VOID_ELEMENTS.has(htmlNode.tagName.toLowerCase()) || htmlNode.location.endTag != null) {
```

Comment:
Do we warn on an invalid closing element for a void element? Seems like it'd be good to do so. For example, this should be a warning on the `</input>`

```html
<input><div></div></input>
```

Reply:
I think this is a good idea but maybe something we can add incrementally to the `no-unclosed-tag` diagnostic.

I'll make an issue.